src/PresentationalComponents/Dashboard/_dashboard.scss

```diff
@@ -111,6 +111,7 @@
       }
 
       > * > .pf-c-card {
+        .pf-c-card__body { z-index: 2; }
```

Comment:
Yeah so this was intentional, but we can certainly go about doing it a different way. Right now, I think the `::before` is not allowing focus on the inside of the card.

If you think there's a better way to solve this, I'm down to do it a different way. This was a similar issue to when I was doing the links in the card not working unless I actually used the button component.

Reply:
@ryelo i checked this, its hard to tell what is happening, but instead of updating the entire card body could you just add a z-index of 1 to the svg ? this should fix the problem

Reply:
Yeah that works for me!

Reply:
<img width="598" alt="Screen Shot 2020-04-22 at 8 23 55 AM" src="https://user-images.githubusercontent.com/6640236/79981441-aeae0600-8472-11ea-9b7e-fb87801d97b5.png">

☝️ with a left n right padding of 0  `  <Button variant="plain" aria-label="Action"><InfoIcon /></Button>`